### ISSUE TITLE
Can't install `pos-cli` on npm > 10.3.0 (POS-101)

### ISSUE THREAD

**Issue by z-x** [Platform-OS/pos-cli#626]
## Expected behavior

`npm i -g @platformos/pos-cli` should install `pos-cli` without errors.

## Actual behavior

Trying to `npm i -g @platformos/pos-cli` results in an error from `node-gyp` trying to rebuild `fsevents`. The problem occurs on `npm` above 10.3.0. Downgrading `npm` fixes the problem.

**Comment by z-x** [comment]
Related issues:
Node: https://github.com/nodejs/node/issues/52436
Chokidar: https://github.com/paulmillr/chokidar/issues/1319
Fsevents: https://github.com/fsevents/fsevents/issues/378

**Comment by z-x** [comment]
From what I've investigated, the failing package is `fsevents`. It's currently used as a dependency of `chokidar`, and `chokidar` is used as a dependency of `livereload`.

Removing those fixes the installation issue. I've tried to replace `chokidar` with `@parcel/watcher` but it does not trigger events on folder operations (https://github.com/parcel-bundler/watcher/issues/165). `turbowatch` on the other hand is ESM package so we can't use it as we're stuck in CJS.
